CONTRIBUTING.md

```diff
@@ -107,7 +107,7 @@ Alternatively, if you know the JIRA ticket number of the issue that you are fixi
 
 ### JavaScript development
 
-_*Note*: We use `node==14` in our development environment._
+_*Note*: We use the [latest release of v14 Node.js](https://nodejs.org/download/release/latest-v14.x/) in our development environment._
```

Comment:
Maybe I don't know what I'm talking about here, but there seem to be many different version of 14.x.y and the exact one does seem to make a difference to the package-lock.json. So I thought best to fix on one particular version.

Reply:
From what I know as long as it's v14 it would work for compiling our app, but I agree that specifying the latest version would help align the package-lock file. 

```suggestion
_*Note*: We suggest using the latest version of [Node.js v.14 (<=14.18.0)](https://nodejs.org/download/release/latest-v14.x/) in our development environment._
```

Reply:
Ah, that makes sense. Thanks for explaining @studioswong

Reply:
But why should we be suggesting <=14.18.0 rather than the latest 14.18.3 (which is what I've been using and seems to work ok)?

Reply:
yea good point - yea let's go for <=4.18.3